### PR TITLE
add-even v.1

### PR DESCRIPTION
**Pull request by Lukasz-334**
add-even-test.

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @Lukasz-334

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Lukasz-334 for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
-    return -1;
+int addEven(const std::vector<int>& numbers)
+{
+    int sum=0;
+    
+    for(auto x:numbers)
+    {
+   	 if(x%2==0)
+    	{
+    	   sum+=x;
```
Comment (gyrydymptum) [inline homework/add-even/addEven.hpp:12]: Add space between operations sum += x

**Comment by ziobron** [comment]
Thanks @gyrydymptum for Code Review.
🏅 1 XP granted

**Pull request closed without merging**